### PR TITLE
Doc page for Textarea component

### PR DESCRIPTION
**Pull request by GomezIvann**

### CODE REVIEW

File: website/screens/components/textarea/usage/TextareaUsagePage.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const TextareaUsagePage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Usage</HeadingLink>
+        <DxcList>
+          <DxcText>
+            Use the texarea when you need users to enter a text longer than a
+            single line
```
Comment (marcialfps) [inline website/screens/components/textarea/usage/TextareaUsagePage.tsx:13]: Please add full stops to sentences.

File: website/screens/components/textarea/specs/TextareaSpecsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+const TextareaSpecsPage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Figure caption="Component design specifications">
+          <Image src={specsImage} alt="Component design specifications" />
+        </Figure>
+        <DxcText as="p">
+          The textarea <Code>color</Code>, <Code>typography</Code>,
```
Comment (marcialfps) [inline website/screens/components/textarea/specs/TextareaSpecsPage.tsx:26]: Add space after `typography`.

File: website/screens/components/textarea/specs/TextareaSpecsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+          <Code>border</Code>, <Code>width</Code> and <Code>margin</Code>{" "}
+          specifications are inherited from the text input, for reference{" "}
+          <DxcLink
+            text="check the text input component documentation"
+            href=""
```
Comment (marcialfps) [inline website/screens/components/textarea/specs/TextareaSpecsPage.tsx:31]: `href="/components/text-input"`

**Review by marcialfps — CHANGES_REQUESTED** [review]

File: website/screens/components/textarea/usage/TextareaUsagePage.tsx (hunk 1 above)
Comment (GomezIvann) [reply website/screens/components/textarea/usage/TextareaUsagePage.tsx:13]: done!

File: website/screens/components/textarea/specs/TextareaSpecsPage.tsx (hunk 2 above)
Comment (GomezIvann) [reply website/screens/components/textarea/specs/TextareaSpecsPage.tsx:26]: done!

File: website/screens/components/textarea/specs/TextareaSpecsPage.tsx (hunk 3 above)
Comment (GomezIvann) [reply website/screens/components/textarea/specs/TextareaSpecsPage.tsx]: done!

**Review by marcialfps — APPROVED** [review]

**Pull request merged**